GameServerLib/Scripting/CSharp/BuffGameScriptController.cs

```diff
@@ -29,7 +30,7 @@ public BuffGameScriptController(Game game, IObjAiBase unit, string buffNamespace
 
             if (_duration >= 0)
             {
-                ApiFunctionManager.CreateTimer(_duration, DeactivateBuff);
+                _remainingDuration = _duration * 1000;
```

Comment:
Unnecessary variable.

Reply:
Why is it unnecessary? It's just to keep track of remaining time and having the full duration of buff stored somewhere else

Reply:
But you can just use `_duration` for it.

Reply:
I think it would be preferable to have the remaining duration be accessible (or at least readable) by scripts. Not sure if it would be for another PR, so mentioning it here.

Reply:
Nevermind, IBuff has TimeElapsed already, which can be used to find timeleft.

Reply:
Seems like there is too much mess with these variables, let's decide what class handle what.